localstack/services/ssm/ssm_listener.py

```diff
@@ -19,12 +19,44 @@ def normalize_name(param_name):
 
 def get_secrets_information(name, resource_name):
     client = aws_stack.connect_to_service('secretsmanager')
-    secret_info = client.get_secret_value(SecretId=resource_name)
-    del secret_info['ResponseMetadata']
-
-    result = {'Parameter': {'SourceResult': secret_info, 'Name': name, 'Value':
-              secret_info.get('SecretString'), 'Type': 'SecureString',
+    try:
+        secret_info = client.get_secret_value(SecretId=resource_name)
+        del secret_info['ResponseMetadata']
+        result = {'Parameter': {'SourceResult': secret_info, 'Name': name, 'Value':
+                secret_info.get('SecretString'), 'Type': 'SecureString',
                             'LastModifiedDate': secret_info.get('CreatedDate')}}
+        return result
+    except client.exceptions.ResourceNotFoundException:
+        return None
+
+
+def has_secrets(names):
+    hasSecrets = False
+    for name in names:
+        if name[0:29] == '/aws/reference/secretsmanager':
+            hasSecrets = True
+            break
+    return hasSecrets
+
+
+def get_params_and_secrets(names):
+    ssmClient = aws_stack.connect_to_service('ssm')
```

Comment:
nitpick: let's better use `snake_case` for variable names, as in the rest of the code base (i.e., `ssm_client`).